src/Models/Block.php

```diff
@@ -38,8 +38,10 @@ protected static function boot()
 
     public function updateSlot()
     {
-        /** @var Slot $slot */
-        $slot = Slot::find($this->slot_id);
+        /** @var Model $slotModel */
+        $slotModel = app('slot');
```

Comment:
Leave it the way it was and just import the Slot model from tipoff/bookings at the top of the Block model